Sources/DistributedSystem/ChannelHandler.swift

```diff
@@ -60,7 +60,7 @@ class ChannelHandler: ChannelInboundHandler {
     }
 
     func errorCaught(context: ChannelHandlerContext, error: Error) {
-        logger.error("\(String(describing: context.remoteAddress)): network error: \(error)")
+        logger.debug("\(String(describing: context.remoteAddress)): network error: \(error)")
```

Comment:
Hmm, do you think this information is not important in troubleshooting?
Maybe "info" ?

Reply:
Good question. Usually, such an error results in a connection breakdown, which DS will attempt to restore in any case. So, the actual error message is not very important.

Reply:
Yep, but there could be some side-effects -- delays, caches becomes stale etc.
And having this logs can help to understand the reason of them.

Reply:
Well, if you think it could help you in the future we can log it on the INFO level.